src/components/scenes/Loans/LoanDashboardScene.js

```diff
@@ -145,15 +138,7 @@ export const LoanDashboardScene = (props: Props) => {
     </TouchableOpacity>
   )
 
-  if (borrowInfosError != null)
-    return (
-      <SceneWrapper background="theme" hasTabs={false}>
-        <SceneHeader underline title={s.strings.loan_dashboard_title} />
-        <Alert title={s.strings.loan_error_title} type="error" message={translateError(borrowInfosError)} />
-      </SceneWrapper>
-    )
-
-  if (borrowInfos == null) {
+  if (loanAccounts == null || !isWalletsLoaded) {
```

Comment:
Yes, temporarily. They eventually load but it takes time. It could be related to the issue RJ was observing where an account with many wallets takes some time to load...